modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -29,14 +37,23 @@ def show
 
         private
 
-        def set_submission_attributes
-          @appeal ||= AppealsApi::NoticeOfDisagreement.find_by(id: params[:nod_id])
-          raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @appeal
+        def set_notice_of_disagreement
+          @nod_id ||= params[:nod_id]
+          raise Common::Exceptions::ParameterMissing, 'nod_id' unless @nod_id
 
+          @notice_of_disagreement ||= AppealsApi::NoticeOfDisagreement.find_by(id: @nod_id)
+          raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @notice_of_disagreement
+        end
+
+        def set_veteran_identifier
+          @veteran_id ||= params[:headers]['X-VA-Veteran-SSN']
+        end
+
+        def set_submission_attributes
           @submission_attributes ||= {
             source: request.headers['X-Consumer-Username'],
-            supportable_id: params[:nod_id],
-            supportable_type: 'AppealsApi::NoticeOfDisagreement'
+            supportable_id: @nod_id,
+            supportable_type: 'NoticeOfDisagreement'
```

Comment:
This will cause loading errors when trying to retrieve this record from the db, since it's not namespaced properly. It should be `AppealsApi::NoticeOfDisagreement`, but if you use `supportable: @notice_of_disagreement` instead, it should load the properly namespaced type for you :)

Reply:
Oh yeah! Sorry, wires crossed with supportable type vs appeal type